include/svs/lib/preprocessor.h

```diff
@@ -159,3 +162,12 @@ inline constexpr bool have_avx512_avx2 = true;
 #endif
 
 } // namespace svs::arch
+
+namespace svs {
+// Maximum values used as default initializers
+inline constexpr size_t UNSIGNED_INTEGER_PLACEHOLDER = std::numeric_limits<size_t>::max();
+inline constexpr float FLOAT_PLACEHOLDER = std::numeric_limits<float>::max();
+inline constexpr float GRAPH_MAX_DEGREE_DEFAULT = 32;
```

Comment:
Can you please look at the docs corresponding to the PR and confirm these `*_DEFAULT` values are visible? Users should be able to see the default value without reading the code. Ideally, there should be numbers visible in the `VamanaBuildParameters` doc [as currently](https://intel.github.io/ScalableVectorSearch/python/vamana.html#svs.VamanaBuildParameters.__init__); alternatively, there should be separate doc entries for the `*_DEFAULT` consts.